app/controllers/api/v1/invoices_controller.rb

```diff
@@ -27,7 +27,7 @@ def show
 
       def index
         invoices = current_organization.invoices
-        invoices = invoices.where(customer_id: params[:customer_id]) if params[:customer_id]
+        invoices = invoices.joins(:customer).where(customers: { external_id: params[:external_customer_id] }) if params[:external_customer_id]
```

Comment:
Style/IfUnlessModifier: Modifier form of `if` makes the line too long. (https://rubystyle.guide#if-as-a-modifier)